Parsers/CommonSecurityLogs-AdditionalExtensionParser.txt

```diff
@@ -0,0 +1,23 @@
+// KQL CommonSecurityLogs AdditionalExtensions Parser
+// Last Updated Date: Aug 14, 2020
+//
+// Parser Notes: 
+// 1. The AdditionalExtensions field generally captures additional fields which are not parsed/mapped to fields by respective vendor CEF configurations
+// 2. The field will have dynamic set of key value pairs separated by ; in the format of the key-balue;key=value;key=value
+// e.g. AdditionalExtensions = deviceTranslatedPort=60095;tunnelType=IPSEC;dnat=No
+// 3. The function takes these dynamic key value pairs and converts into json for easier accessing.
```

Comment:
"converts them into json".<stop>  no need to include the "for easier accessing" part.